lib/ansible/constants.py

```diff
@@ -135,6 +135,7 @@ def set_constant(name, value, export=vars()):
     scp_extra_args=('ansible_scp_extra_args', ),
     ssh_extra_args=('ansible_ssh_extra_args', ),
     ssh_transfer_method=('ansible_ssh_transfer_method', ),
+    pkcs11_provider=('ansible_pkcs11_provider', ),
```

Comment:
again, you dont need this, just the part in the connection plugin

Reply:
Thanks! I was using ansible_ssh_extra_args as my example, and wasn't completely clear if this was needed or not.